cmds/core/netcat/netcat.go

```diff
@@ -87,8 +87,12 @@ func evalParams(args []string, f flags) (*netcat.Config, error) {
 	// The port is optional as the second argument.
 	switch config.ConnectionMode {
 	case netcat.CONNECTION_MODE_CONNECT:
+		if f.sourcePort == "" {
+			f.sourcePort = netcat.DEFAULT_SOURCE_PORT
+		}
+
 		if len(args) < 1 {
-			return nil, fmt.Errorf("missing host")
+			return nil, fmt.Errorf("%w: missing host", os.ErrInvalid)
```

Comment:
I have gotten asked, in reviews, to put the %w last, as it is a convention. Looking at most Go source, I see that is true. I am wondering why you put it first in all these errors?

Reply:
I can change the error output to be conventional. But I have to say that `invalid argument: missing host` instead of `missing host: invalid argument` seems more readable.